### PR TITLE
provide the previous filename in file_renamed_alert

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

File: src/alert.cpp [hunk 1]
```diff
@@ -247,14 +248,22 @@ namespace libtorrent {
...
+		return m_alloc.get().ptr(m_old_name_idx);
+	}
+
 	std::string file_renamed_alert::message() const
 	{
 		std::string ret { torrent_alert::message() };
 		char msg[200];
-		std::snprintf(msg, sizeof(msg), ": file %d renamed to "
-			, static_cast<int>(index));
+		std::snprintf(msg, sizeof(msg), ": file %d renamed from \"",
```
Comment (aldenml) [inline src/alert.cpp]: not an issue by any mean, but are you moving to trailing `,`?

Comment (arvidn) [reply src/alert.cpp]: unintentional :)

Comment (aldenml) [reply src/alert.cpp]: 😀

**Comment by an0n666** [comment]
arvidn can you please backport this to RC1_2?

**Pull request merged**